tests/integration/debug/cmd_debug_test.go

```diff
@@ -40,14 +40,15 @@ var _ = Describe("odo debug command serial tests", func() {
 		os.Unsetenv("GLOBALODOCONFIG")
 	})
 
-	It("should auto-select a local debug port when the given local port is occupied", func() {
+	It("check that machine output debug information works", func() {
 		helper.CopyExample(filepath.Join("source", "nodejs"), context)
+
 		helper.CmdShouldPass("odo", "component", "create", "nodejs:latest", "nodejs-cmp-"+project, "--project", project, "--context", context)
 		helper.CmdShouldPass("odo", "push", "--context", context)
 
 		stopChannel := make(chan bool)
 		go func() {
-			helper.CmdShouldRunAndTerminate(60*time.Second, stopChannel, "odo", "debug", "port-forward", "--context", context)
+			helper.CmdShouldRunAndTerminate(60*time.Second, stopChannel, "odo", "debug", "port-forward", "--local-port", "9999", "--context", context)
```

Comment:
Travis is failing because 9999 port is occupied and thus the fakePortListener below doesn't start. You need to first find a free port like in the other non-serial parallel test cases.